### ISSUE TITLE
Add option to force E2EE

### ISSUE THREAD

**Issue by heikar** [jitsi/jitsi-meet#7920]
According to #7919, please add a function so the moderator can force the E2EE.
Because in a lot of meetings it is important, that the E2EE is always active.

**Comment by timomayer** [comment]
Would really muc love to see this feature in one of the next releases, any chance for that :)?

**Comment by saghul** [comment]
We have no plans to do this in the short term sorry.

That said, patches implementing such functionality would be very welcome.

**Comment by koepferd** [comment]
We are considering contributing here, yet we have no knowledge of the codebase. Can you estimate how much time the implementation of this would take?

Kind regards 🙂

**Comment by saghul** [comment]
I think it could be implemented reasonably easily now that we've made some changes.

Right now, if a participant enables E2EE it will be enabled for everyone else automatically.

So having an option to start in E2EE mode in the prejoin screen would do.

That option would do the following:

- note the desired muted states
- mute audio and video
- enable E2EE upon join
- restore the desired muted states

I just typed this while having my morning ☕ so there is a chance some more details need to be taken care of :-)
